### DISCUSSION TITLE
I got this wierd symbol before ➜ character.

### DISCUSSION THREAD

**Discussion by zer09** [starship/starship#1486] [Q&A] [answered]
Hello good day,
What is this symbol before the `➜` character?

![Screenshot_20200716_215337](https://user-images.githubusercontent.com/780780/87680943-9d923e00-c7b0-11ea-8e36-a66d8b465c09.png)

**Comment by johnletey** [comment, marked as answer]
@zer09 It's the [jobs module](https://starship.rs/config/#jobs) character, which is a module that shows the current number of background jobs running.

**Reply by zer09** [reply]
But I dint run any background job, when I tried `bg` the result is `bash: bg: current: no such job` or with `jobs` will only show empty prompt.

That will only show if I'm on Manjaro KDE, but not when I'm on Pop-OS.

**Reply by johnletey** [reply]
@zer09 I can see you're using the `bash` shell. We don't use the `bg` command to determine background jobs. Try running the following:

```bash
echo $(jobs -p | wc -l)
```

👆 This command should list out the number of current background jobs running on your machine.

**Reply by zer09** [reply]
Thanks @jletey I tried the command and this is the result.

![Screenshot_20200717_154942](https://user-images.githubusercontent.com/780780/87762351-979a6c80-c845-11ea-9531-057d66871efc.png)

Is that intentional or that's a bug? It's reporting I have two jobs running even I only run only one background job. And the OS is just freshly installed and the jobs symbol show right away.

BTW this is my config

```toml
[character]
symbol = "➜"
error_symbol = "✗"
use_symbol_for_status = true

[git_branch]
symbol = "🌱 "

[memory_usage]
disabled = false
```

**Reply by johnletey** [reply]
@zer09 Hmm this is very strange indeed. Did you install the latest stable release? Or did you compile from master?

It might have something to do with the new format strings 🤷‍♂️

**Reply by zer09** [reply]
I installed it from AUR https://aur.archlinux.org/packages/starship/ and looking into pkgbuild It's look like from stable release `pkgver='0.44.0'`

**Reply by andytom** [reply]
This might be the same issue as #972 where there is a background job that is running when starship draws the prompt but it has completed by the time you check the jobs.

**Reply by zer09** [reply]
@andytom Thanks! yeah it is, and I follow this https://github.com/starship/starship/issues/972#issuecomment-652814709, and it seems to fix the issue.

**Comment by santiestrada8585-collab** [comment]
Si
